chrome/content/zotero/elements/collapsibleSection.js

```diff
@@ -106,6 +106,19 @@
 			this.setAttribute('summary', val);
 		}
 		
+		get notCollapsible() {
```

Comment:
Yes, exactly. My original thought was that "not collapsible" is a special case, so it should have special naming. It can end up being a bit confusing, especially if we wanted to check if something is collapsible because then it had to be a double-negative.

So I changed it to be `collapsible` instead, that's definitely better